digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/dataimport/CsvDataImporter.scala

```diff
@@ -110,26 +119,73 @@ class TrafficSignCsvImporter extends CsvDataImporterOperations {
   }
 
   private def verifyValueCode(parameterName: String, parameterValue: String): ParsedAssetRow = {
-    if(parameterValue.forall(_.isDigit) && TRTrafficSignType.apply(parameterValue.toInt).source.contains("CSVimport")){
+    if(parameterValue.forall(_.isDigit) && TrafficSignType.applyTRValue(parameterValue.toInt).source.contains("CSVimport")){
       (Nil, List(AssetProperty(columnName = codeValueFieldMappings(parameterName), value = parameterValue.toInt)))
     }else{
       (List(parameterName), Nil)
     }
   }
 
-  def tryToInt(propertyValue: String ) = {
+  def getRightRoadLinkUsingBearing(assetBearing: Option[Int], assetCoordinates: Point): (Seq[VVHRoadlink], Seq[RoadLink]) = {
+    val toleranceInDegrees = 25
```

Comment:
VVRoadLink and RoadLink are the same  **VVHRoadlink after enrich is a RoadLink**

shouldn't be use both as return